tests/integration/test_charm.py

```diff
@@ -36,7 +41,111 @@ async def test_build_and_deploy(ops_test: OpsTest):
         raise_on_blocked=True,
         timeout=1000,
     )
+    assert len(ops_test.model.applications[APP_NAME].units) == 1
     assert ops_test.model.applications[APP_NAME].units[0].workload_status == "active"
 
-    # effectively disable the update status from firing
-    await ops_test.model.set_config({"update-status-hook-interval": "60m"})
+
+@pytest.mark.skip_if_deployed
+@pytest.mark.abort_on_fail
+async def test_scale_to_ha(ops_test: OpsTest):
+    """Scale the charm to HA."""
+    await ops_test.model.applications[APP_NAME].scale(len(UNIT_IDS))
+
+    # Blocks until all units are in active state
+    logger.info("Wait for app active status")
+    sleep(10)
+    await ops_test.model.wait_for_idle(
```

Comment:
suggestion from raul on my PR: could do `await ops_test.model.block_until(lambda: len(ops_test.model.applications[APP_NAME].units) == len(UNIT_IDS))`

Reply:
Want to refactor/unify tests with machine, so not worrying to much with this right now.